doc/userdoc/guides/nest2_to_nest3/nest3_features/parametrization.rst

```diff
@@ -406,6 +406,28 @@ statement. Three arguments are required:
 .. image:: ../../../static/img/NEST3_26_0.png
 
 
+.. _compare_ex:
+
+Compare parameters
+^^^^^^^^^^^^^^^^^^
+
+In the ``nest.logic.conditional()`` function above we used a comparison between a
+``nest.Parameter`` and a value. It is also possible to compare one ``nest.Parameter``
+with another, and all comparison operators are supported. The result of such comparisons
+is a new ``nest.Parameter``, which evaluates to either 1 or 0 for true and false,
+respectively. The resulting comparison ``nest.Parameter`` can be used in a ``nest.logic.conditional()``,
+but also without it if the desired result is zero where the comparison is false.
+
+::
+
+    # As an example, take a step function where the resulting value is
+    # 0.5 for positive values, and zero for negative.
+    nest.logic.conditional(nest.random.uniform(min=-1., max=1.) > 0., 0.5, 0.)
```

Comment:
```suggestion
    nest.logic.conditional(nest.random.uniform(min=-1., max=1.) > 0.0, 0.5, 0.0)
```

I know you don't have to add a zero after the decimal, but I think it looks strange especially being followed by a comma